front/src/modules/simulationResult/components/SpaceTimeChart/SpaceTimeChart.tsx

```diff
@@ -211,20 +216,43 @@ export default function SpaceTimeChart(props: SpaceTimeChartProps) {
     redrawChart();
   }, [resetChart, rotate, selectedTrain, trainSimulations, height]);
 
+  /* redraw the trains if the time scale range has changed from Timeline */
+  useEffect(() => {
+    if (chart && timeScaleDomain && timeScaleDomain.source !== 'SpaceTimeChart') {
+      const currentTimeRange = timeScaleDomain.range;
+      if (currentTimeRange) {
+        const newChart = { ...chart };
+        newChart.x.domain(currentTimeRange);
+        redrawChart();
+      }
+    }
+  }, [timeScaleDomain]);
+
   /* add behaviour on zoom and mousemove/mouseover/wheel on the new chart each time the chart changes */
   useEffect(() => {
-    if (trainSimulations && selectedTrain) {
-      const dataSimulation = createTrain(CHART_AXES.SPACE_TIME, selectedTrain as Train);
-      enableInteractivity(
-        chart,
-        dataSimulation,
-        CHART_AXES.SPACE_TIME,
-        rotate,
-        setChart,
-        simulationIsPlaying,
-        updateTimePosition,
-        timeScaleRange
-      );
+    if (chart) {
+      const newTimeScaleRange = (rotate ? chart.y.domain() : chart.x.domain()) as [Date, Date];
+
+      if (setTimeScaleDomain) {
+        setTimeScaleDomain({
+          range: newTimeScaleRange,
+          source: 'SpaceTimeChart',
+        });
+      }
+
+      if (trainSimulations && selectedTrain && timeScaleDomain) {
```

Comment:
With this, we lose the possibility to zoom on the GET on stdcm. Do we want that ?